### ISSUE TITLE
Adding Google image credits

### ISSUE THREAD

**Issue by MAPlanGo** [nextgis/quickmapservices#140]
Love QMS plug in but was wondering about a possible enhancement. How easy would it be to add Google image source credit?

**Comment by simgislab** [comment]
you mean this?

![](http://m-d.me/img/ss/20171006-TboWmKfp.png)

**Comment by MAPlanGo** [comment]
Thanks very much for looking into it. It certainly looks like that would do the trick, except that Google credits are not present, and nor do they appear bottom right after ticking box. Is it a bug?

![image](https://user-images.githubusercontent.com/17493953/31316102-a213a52a-ac26-11e7-881c-336f85537f7c.png)

**Comment by simgislab** [comment]
No, it is not a bug. It is somebody who added this service didn't add credits so there is nothing to show up. 

You can fix it.

Go to qms.nextgis.com and create a copy of the service you need with all missing details.

**Comment by MAPlanGo** [comment]
Thanks for getting back to me. I've been looking into it and i cannot work out how to add the google attribution. It would need to update dynamically depending on where the user is viewing and i suspect that makes it more difficult (and perhaps explains why it wasn't added originally). I'm afraid it's beyond me but I'm sure it would benefit many users if anyone out there can make this addition by making it much easier for users to comply with google terms/ attribution guidelines: https://www.google.com/permissions/geoguidelines/attr-guide.html

**Comment by simgislab** [comment]
Yes, I believe dynamically determining if the copyright info should be shown based on extent is beyond the scope of QMS, but within scope of QGIS itself.  We can have a look when resources are available.

Meanwhile, you can send feedback to service author suggesting changes to the service using Feedback option on http://qms.nextgis.com.
